src/Components/Config/configHook.tsx

```diff
@@ -235,3 +235,19 @@ export function useConfig<T>(name: string, defaultValue?: T): T {
 
   return config[name] as T;
 }
+
+export function useLocalizedLink(configKey: 'privacy_policy' | 'consent_to_contact') {
+  const intl = useIntl();
+  const locale = intl.locale as string;
+  const links = useConfig<Partial<Record<string, string>>>(configKey, {});
+
+  return links[locale] || links['en-us'] || '';
```

Comment:
Nit: I like `??` more because it does not have unexpected behavior for `0` and `''`
```suggestion
  return links[locale] ?? links['en-us'] ?? '';
```